packages/camera/example/android/gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
Don't check this into the flutter/plugins repo. Add to .gitignore.

Reply:
Done